server/cli/launch.py

```diff
@@ -50,7 +52,7 @@
     show_default=True,
     help="Open the web browser after launch.",
 )
-@click.option("--port", "-p", help="Port to run server on.", metavar="", default=5005, show_default=True)
+@click.option("--port", "-p", help="Port to run server on.", metavar="", show_default=True)
```

Comment:
Also, consider making the help text indicate that the app will attempt to find an open port if this is not specified.

Reply:
will fix